syntaxes/jvm-bytecode.json

```diff
@@ -0,0 +1,395 @@
+{
+  "$schema": "./schema/tmlanguage.json",
```

Comment:
The schema this references is from https://github.com/martinring/tmlanguage.
You will have to add its license to NOTICE.md too if you want to include it in this repository.

Alternatively, you can just remove the schema file and the `$schema` field from this and the other syntax and it would work fine. I only included it in my original project because it makes it easier to edit the syntax.

Reply:
Thanks for the heads-up @mnxn